docs/usage.rst

```diff
@@ -235,6 +235,8 @@ Additional command line options provided by django_behave:
 ``--use-existing-database``
 ***************************
 
+(Depricated) Please use ``--testrunner behave_django.runner.ExistingDatabaseTestRunner`` instead.
```

Comment:
Should spell "Deprecated" ([dictionary](https://www.dictionary.com/browse/deprecated))